From 2ad0f8eb65d7af9fcaa79d55f266ad653694b177 Mon Sep 17 00:00:00 2001
From: Julien Massot <julien.massot@iot.bzh>
Date: Tue, 1 Jun 2021 10:38:56 +0200
Subject: [PATCH 3/3] dts: r8a77990: add optee node

Signed-off-by: Julien Massot <julien.massot@iot.bzh>
---
 arch/arm64/boot/dts/renesas/r8a77990.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990.dtsi b/arch/arm64/boot/dts/renesas/r8a77990.dtsi
index a12214b0aae0..53cf4f5b962c 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77990.dtsi
@@ -127,6 +127,13 @@ CPU_SLEEP_0: cpu-sleep-0 {
 		};
 	};
 
+	firmware {
+		optee {
+			compatible = "linaro,optee-tz";
+			method = "smc";
+		};
+	};
+
 	extal_clk: extal {
 		compatible = "fixed-clock";
 		#clock-cells = <0>;
-- 
2.31.1

