diff --git a/plat/renesas/common/bl2_secure_setting.c b/plat/renesas/common/bl2_secure_setting.c
index 8367b2270..2cef3369b 100644
--- a/plat/renesas/common/bl2_secure_setting.c
+++ b/plat/renesas/common/bl2_secure_setting.c
@@ -22,9 +22,9 @@ static const struct {
 	 * LIFEC0 (SECURITY) settings
 	 * Security attribute setting for master ports
 	 * Bit 0: ARM realtime core (Cortex-R7) master port
-	 *        0: Non-Secure
+	 *        1: Secure
 	 */
-	{ SEC_SRC, 0x0000001EU },
+	{ SEC_SRC, 0x0000001FU },
 	/*
 	 * Security attribute setting for slave ports 0 to 15
 	 *      {SEC_SEL0,              0xFFFFFFFFU},
