From a2130b1ec9e703931d53d2483751051ef49129f6 Mon Sep 17 00:00:00 2001
From: Julien Massot <julien.massot@iot.bzh>
Date: Wed, 21 Oct 2020 17:50:59 +0200
Subject: [PATCH] hack: do not check for reserved memory

---
 fs/fs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/fs.c b/fs/fs.c
index 0c66d60477..9e42098932 100644
--- a/fs/fs.c
+++ b/fs/fs.c
@@ -717,7 +717,7 @@ int do_load(cmd_tbl_t *cmdtp, int flag, int argc, char * const argv[],
 			(argc > 4) ? argv[4] : "");
 #endif
 	time = get_timer(0);
-	ret = _fs_read(filename, addr, pos, bytes, 1, &len_read);
+	ret = _fs_read(filename, addr, pos, bytes, 0, &len_read);
 	time = get_timer(time);
 	if (ret < 0)
 		return 1;
-- 
2.26.2

