diff --git a/drivers/clk/renesas/renesas-cpg-mssr.c b/drivers/clk/renesas/renesas-cpg-mssr.c
index 0f9a4abe5a..3d75e21248 100644
--- a/drivers/clk/renesas/renesas-cpg-mssr.c
+++ b/drivers/clk/renesas/renesas-cpg-mssr.c
@@ -128,10 +128,12 @@ int renesas_clk_remove(void __iomem *base, struct cpg_mssr_info *info)
 
 		if (info->reg_layout == CLK_REG_LAYOUT_RCAR_V3U)
 			continue;
-
+#ifdef RCAR_GEN3_CA57_ONLY
+		/* U-Boot should not touch the Realtime clocks */
 		clrsetbits_le32(base + RMSTPCR(i),
 				info->mstp_table[i].rdis,
 				info->mstp_table[i].ren);
+#endif
 	}
 
 	return 0;
